fontforge/splineutil.c

```diff
@@ -1945,6 +1945,7 @@ SplinePointList *SplinePointListTransformExtended(SplinePointList *base, real tr
 	    int i;
 	    for ( i=0; i<spl->spiro_cnt-1; ++i )
 		TransformSpiro(&spl->spiros[i], transform);
+		SSRegenerateFromSpiros(spl);
```

Comment:
Was this omission breaking all transformations of spiros?

Reply:
It was making them look really weird in the UI; not complete breakage, but close.

Reply:
(We can see it was simply forgotten as it's used elsewhere, e.g. in function SplinePointListSpiroTransform, splineutil.c:2023)